inject/src/main/java/io/micronaut/context/DefaultApplicationContext.java

```diff
@@ -200,7 +209,9 @@ protected void registerConversionService() {
     public synchronized @NonNull
     ApplicationContext stop() {
         ApplicationContext stop = (ApplicationContext) super.stop();
-        environment = null;
```

Comment:
I removed setting the environment to null here. Otherwise, if someone calls context.stop() and context.getEnvironment, a new environment is getting created, which is not right.